packages/remediations/src/NewRemediationWizard.js

```diff
@@ -34,93 +36,44 @@ function createNotification (id, name, isNewSwitch) {
     };
 }
 
-class RemediationWizard extends Component {
-
-    state = {
-        open: false,
-        schema: {},
-        errors: []
-    }
+const initialState = {
+    open: false,
+    schema: {},
+    errors: []
+};
 
-    constructor() {
-        super();
-        this.container = React.createRef(document.createElement('div'));
+function reducer(state, action) {
+    switch (action.type) {
+        case 'update':
+            return {
+                ...state,
+                ...action.payload
+            };
```

Comment:
Not sure if using reducer like this makes sense when there is just on action called 'update'.

You can achieve the same by using:

```jsx
const [state, setState] = useState({ ... });

...

setState({ ...state, newValues })
```

Or please split the reducer to more actions. For example instead of 

`const closeWizard = () => dispatch({ type: 'update', payload: { isOpen: false } });`

create a new action

```jsx
        case 'close':
            return {
                ...state,
                isOpen: false
            };
```

`const closeWizard = () => dispatch({ type: 'close' });`